vtex-apps/checkout-cart
# Display messages from API in toasts

#### What problem is this solving?

This displays messages with general context that come from the Checkout API in toasts on the Cart page. If the OrderForm contains multiple messages, they are displayed in sequence.

#### How should this be manually tested?

- Use [this workspace](https://toasts--vtexgame1.myvtex.com).
- Add a [random item](https://toasts--vtexgame1.myvtex.com/produto-simples-2/p), go to `/checkout` and set your CEP to `22250-040`.
- Add [this item](https://toasts--vtexgame1.myvtex.com/polera-de-futbol-nike-manga-corta-20/p) and [this item](https://toasts--vtexgame1.myvtex.com/polera-de-futbol-nike-manga-corta-20/p) to your cart. This triggers two messages: "O tipo de entrega foi alterado" and "O valor do frete foi alterado".
- Go to `/cart` and verify both toasts appear in sequence.

#### Checklist/Reminders

- [x] [Linked this PR to a Clubhouse story (if applicable)](https://app.clubhouse.io/vtex/story/20914/implementar-toasts).

#### Screenshots or example usage

#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements 

#### Notes
